config/packages/Centreon.yaml

```
@@ -226,6 +226,7 @@ services:
 
     Core\Platform\Application\Repository\ReadUpdateRepositoryInterface:
         class: Core\Platform\Infrastructure\Repository\FsReadUpdateRepository
+        arguments: ['%centreon_install_path%']
```

I guess you could also use "named arguments" to make it clearer in the configuration file
```suggestion
        arguments:
            $installDir: '%centreon_install_path%'
```